services/rulesValidator.js

```diff
@@ -56,6 +56,16 @@ const getRuleMethodName = (ruleName) => {
     return name;
 };
 
+const getParentNodeTagname = (xml, xpathContext) => {
+    const parser = new DOMParser();
+    const xmlDoc = parser.parseFromString(xml, 'text/xml');
+    const elementNode = select(xpathContext.xpath, xmlDoc).find(
+        (element) => element.lineNumber === xpathContext.lineNumber
+    );
+    const parentElement = elementNode.parentNode;
+    return parentElement.tagName;
```

Comment:
Could you please change this to:
```
return `${parentElement.tagName}/${elementNode.tagName}`
```

That way it returns like `transaction/description` instead of just `transaction`.